include/RAJA/index/IndexSetBuilders.hpp

```diff
@@ -81,7 +81,8 @@ void buildTypedIndexSetAligned(IndexSet& hiset,
  *
  ******************************************************************************
  */
-void buildLockFreeBlockIndexset(IndexSet& iset,
+void buildLockFreeBlockIndexset(RAJA::TypedIndexSet<RAJA::RangeSegment,
```

Comment:
This one uses RangeSegments only.

Reply:
Rather than deal with these changes here, shall I move the indexsetbuilders into LULESH?

Reply:
No. They need work, but I believe folks are using them.

Reply:
Changing them to use strongly-typed IndexSets with a subset of segment types will break for existing users.

Reply:
Good point. Leave them as is for now. I don't think there is any additional overhead for not using segment types.

Reply:
👍